plugins/processors/ifname/ifname_test.go

```diff
@@ -9,88 +9,87 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/influxdata/telegraf/config"
-	"github.com/influxdata/telegraf/internal"
-	"github.com/influxdata/telegraf/internal/snmp"
 	si "github.com/influxdata/telegraf/plugins/inputs/snmp"
-	"github.com/influxdata/telegraf/testutil"
 )
 
-func TestTable(t *testing.T) {
-	if testing.Short() {
-		t.Skip("Skipping integration test in short mode")
```

Comment:
I wrote this test and meant for the snmp agent to be set up manually, not with docker-compose like the others.  Sorry not to add a comment to make this clear.  Since it's not set up to run with docker-compose we shouldn't add the Integration suffix to the func name.

Reply:
No problem, I've removed the Integration from the end of the name and added a skip for until it's fixed.